README.md

```diff
@@ -24,6 +24,7 @@ The simulation currently consists of just one very broad class, the LTF Array Si
  * `atf`: use *Arbiter Threshold Functions*, that is, transform the challenges in a way such that we simulate physical implementations of Arbiter PUFs
  * `mm`: experimental input transformation with long PTF representation
  * `lightweight_secure`: input transformation as defined by Majzoobi et al. in [Lightweight Secure PUFs](http://aceslab.org/sites/default/files/Lightweight%20Secure%20PUFs_0.pdf)
+ * `shift_lightweight_secure`: as defined by Majzoobi et al., but with the shift operation executed first.
```

Comment:
I still believe that this is the intended transformation by Majzoobi et al. @benj-zen